### ISSUE TITLE
Pull request 75

### ISSUE THREAD

**Issue by mhines-usgs** [USGS-R/grantools#76]
https://github.com/USGS-R/grantools/pull/75
and
https://github.com/USGS-R/grantools/pull/77

- [x] source
- [x] windows 3.1
- [x] windows 3.2
- [ ] mavericks 3.1
- [ ] mavericks 3.2
- [x] win binaries to AWS
- [ ] osx binaries to AWS

**Comment by jread-usgs** [comment]
Am running things on mavs 3.2 and won't be able to switch for a while. Are any of the hazards users mac users?

**Comment by mhines-usgs** [comment]
@jread-usgs nope, neither Heather S or Justin.

**Comment by jread-usgs** [comment]
@ldecicco-USGS and/or @lawinslow can we get the win bin up on GRAN and circle back for mac?

**Comment by ldecicco-USGS** [comment]
yeah, but @lawinslow , wait on a new pull request from me.

**Comment by lawinslow** [comment]
Ok, waiting....

**Comment by ldecicco-USGS** [comment]
OK, good enough for now. I won't be able to get EGRET up today, so have at it.

**Comment by lawinslow** [comment]
On it

**Comment by lawinslow** [comment]
Windows up

**Comment by mhines-usgs** [comment]
Thanks dudes!

**Comment by lawinslow** [comment]
@jread-usgs @ldecicco-USGS this still open?

**Comment by ldecicco-USGS** [comment]
I haven't moved any mac binaries anywhere recently

**Comment by jread-usgs** [comment]
can we add the first `gsplot` release to this too?

**Comment by lawinslow** [comment]
Sure, tack it on!
